swagger/swagger-generator/generator-core/src/main/java/org/apache/servicecomb/swagger/generator/core/processor/annotation/ApiProcessor.java

```diff
@@ -17,18 +17,69 @@
 
 package org.apache.servicecomb.swagger.generator.core.processor.annotation;
 
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
 import org.apache.servicecomb.swagger.generator.core.ClassAnnotationProcessor;
 import org.apache.servicecomb.swagger.generator.core.SwaggerGenerator;
 import org.springframework.util.StringUtils;
 
 import io.swagger.annotations.Api;
+import io.swagger.models.Swagger;
```

Comment:
Can you add an integration test for this feature?

Reply:
Integration test has been added.
In order to test the consumes, I change `BodyProcessor` in `BodyProcessorCreator.java` to let users can specify request Content-Type(Currently it's `application/json` mandatorily).